protocol/src/main/java/io/camunda/zeebe/protocol/record/value/ProcessInstanceCreationRecordValue.java

```diff
@@ -37,4 +38,15 @@
    * @return the unique key of the BPMN process definition to create a process from
    */
   long getProcessDefinitionKey();
+
+  /**
+   * @return list of start instructions (if available)
+   */
+  List<ProcessInstanceCreationStartInstructionValue> getStartInstructions();
+
+  @Value.Immutable
+  @ImmutableProtocol(builder = ImmutableProcessInstanceCreationStartInstructionValue.Builder.class)
+  interface ProcessInstanceCreationStartInstructionValue {
```

Comment:
❓ Why is this an inner interface? Other similar constructs (like `EvaluatedInputValue` which belongs to `EvaluatedDecisionValue`) are all standalone interfaces. I ask this because I have a feeling that the canonical name of an inner interface (`A.B`) might interfere with the generated code, although I have no real proof of that. I would suggest continuing in the same fashion to favor consistency if there is no specific reason to divert from it. Let me know what you think.

Reply:
Interestingly, it doesn't interfere with code generation (I had the same concern). 

Could be a file of it's own. I am indifferent. Let me know how much you want it.

Reply:
I have no strong opinion either way. Feel free to ignore it